process-1.6.13.2 does not build with Win32-2.3.1.1

process-1.6.13.2 does not build with Win32-2.3.1.1:
```
[4 of 5] Compiling System.Process   ( System\Process.hs, dist\build\System\Process.o )

System\Process.hs:97:44: error:
    Module ‘System.Win32.Process’ does not export ‘getCurrentProcessId’
   |
97 | import System.Win32.Process (getProcessId, getCurrentProcessId, ProcessId)
   |                                            ^^^^^^^^^^^^^^^^^^^
```

--- a/process.cabal
+++ b/process.cabal
@@ -58,7 +58,7 @@ library
         c-sources:
             cbits/win32/runProcess.c
         other-modules: System.Process.Windows
-        build-depends: Win32 >=2.2 && < 2.13
+        build-depends: Win32 >=2.4 && < 2.13
         -- ole32 and rpcrt4 are needed to create GUIDs for unique named pipes
         -- for process.
         extra-libraries: kernel32, ole32, rpcrt4

